content/docs/setup/behavioral_changes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Behavioral Changes in 1.1
+description: Major changes in behavior, both backward compatibility breaking and non, in Istio 1.1.
+weight: 15
+icon: setup
+
+---
+
+# Major Behavioral Changes in Istio 1.1
```

Comment:
No need for this, the site infra fills-in the main title.

Reply:
done